Plugins/TGeo/include/Acts/Plugins/TGeo/TGeoITkModuleSplitter.hpp

```diff
@@ -0,0 +1,106 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Plugins/TGeo/ITGeoDetectorElementSplitter.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+class TGeoNode;
+
+namespace Acts {
+
+class TGeoDetectorElement;
+
+/// @brief TGeoITkModuleSplitter
+///
+/// Split Itk modules into submodules, depending on the sensor type
+class TGeoITkModuleSplitter : public ITGeoDetectorElementSplitter {
```

Comment:
Again here, if this is specific to the 'ITk' I would say that the right location for this file is in `Examples/Detectors/TGeo/` or `Examples/Detectors/ITk` ... in `Acts/Core` we should only have the base class.